### ISSUE TITLE
Pagination a11y

### ISSUE THREAD

**Issue by Heydon** [iTwin/stratakit#1233]
## 1. By default, page options should be rendered as links

Currently, buttons are rendered. This seems to be the case unless [a router is integrated](https://mui.com/material-ui/react-pagination/#router-integration)

## 2. Redundant/repeated label

The label is hard-coded as "pagination navigation" (the "navigation" part is redundant, since that's already derived from the role. The consumer should have control of the label, preferably via a required prop.
 
## 3. current page highlight needs higher contrast ✅

The Non-Text Contrast ratio for the grey circle against the page background is currently just 1.23.
